lockup/README.md

```diff
@@ -1,3 +1,371 @@
 # Lockup / Vesting contract
 
-*Work in progress.*
+## Overview
+
+This contract acts as an escrow that locks and holds owner's funds for the lockup period. The lockup period either starts
+at the given timestamp or from the moment transfers are enabled by voting.
+If transfers are not enabled yet, the contract keeps the account ID of the transfer poll contract.
+When the transfer poll is resolved, it returns the timestamp when it was resolved and it's used as the beginning of the
+lockup period.
+
+Once all funds are unlocked (including vesting) and transfers are enabled, the owner can add a full access key to the
+account. This will allow the owner to turn this account into a regular account, remove the contract, or delete the account
```

Comment:
```suggestion
account. This will allow the owner to turn this account into a regular account, claim the remaining funds, and remove the contract 
or delete the account.
```

The ands and ors were confusing.  Is this what you meant?